cloudapp/src/app/main/main.component.html

```diff
@@ -3,6 +3,7 @@
   <div class="eca-actions">
     <button mat-flat-button color="secondary" (click)="resetOptions()">Reset</button>
     <button type="submit" mat-flat-button color="primary" (click)="onPrint()" [disabled]="!form.valid || loading">Print</button>
+    <button type="submit" mat-flat-button color="primary" (click)="onExcel()" [disabled]="!form.valid || loading">Download Excel</button>
```

Comment:
I'm not sure about having two primary buttons. There's a "rule" about not having two primary buttons. But not sure what the solution is.

Reply:
I agree, there should only be one primary button. As a suggestion I made the 'Download Excel' button a secondary button and also reordered/renamed the buttons to 'Print', 'Excel', 'Reset'.
What do you think?

Reply:
Thank you for this. The Alma UI orders the buttons so that the primary button is on the right and we should follow that. So please change it to «Reset» «Excel» «Print».

In the popup dialog you get with you click Alma's «Print Slip Report» button, this equivalent function to this is labelled «Download». Maybe the label in the app for the same function should be the same? I'll leave this totally up to you.

That same popup dialog has «Download» divider «Cancel» «Send». If you have a mind to, doing something similar here I think would work well: «Reset» divider «Excel/Download» «Print».

Reply:
We should of course stick to the Cloud Apps style guide, you're right! Sadly there is no rule on how to handle three buttons. I think it's a good idea to use a known pattern as in the popup dialog.
I implemented your suggestion: '«Reset» divider «Excel/Download» «Print»' in commit 1593515